minard/views.py

```diff
@@ -1296,6 +1318,33 @@ def pca_run_detail(run_number):
                            run_number=run_number,
                            run=run)
 
+@app.route('/burst_run_detail/<run_number><subrun><sub>')
+def burst_run_detail(run_number, subrun, sub):
+    server = couchdb.Server("https://snoplus:"+app.config["COUCHDB_PASSWORD"]+"@"+app.config["COUCHDB_HOSTNAME"])
+    db = server["burst"]
+
+    for row in db.view('_design/burst/_view/burst_by_run'):
```

Comment:
Do you have an estimate for how many bursts will be created? I'm worried that iterating over every single CouchDB element is going to be very slow when there are a lot of entries. We've already had issues with the DQ pages being very slow to load and I expect that will get worse and worse as we take more data.

Two things I think could be done to fix this:

1. Use the nearline SQL database to store your results
2. Write a couchdb view which returns the documents based on the run

The first would have lots of advantages (working with a SQL database is *much* easier than CouchDB), and means you could do very specific searches, i.e.

```
SELECT * FROM burst WHERE run = [run] and subrun = [subrun] ORDER BY timestamp;
```

but I realize that may be a lot of work if you already have everything set up for CouchDB.

The second option should be pretty easy.

Reply:
Honestly, the number of bursts is purely dependent on how we set the trigger (this is not finalized yet). I think it's quite likely there is going to be a lot of them, especially in the early period when we 'tune' the triggers. 

Didn't know there is an sql nearline database:O

For now, I will create a view that will load specifically for run_subrun_burst so that it will always return just the one document we need.